Utilities/build-script-helper.py

```diff
@@ -137,6 +137,11 @@ def get_swiftpm_options(swift_exec: str, args: argparse.Namespace, suppress_verb
         swiftpm_args += [
             '-Xlinker', '-rpath', '-Xlinker', '$ORIGIN/../lib/swift/android',
         ]
+    elif '-freebsd' in build_target:
+        # pkg installs packages to /usr/local/include on FreeBSD
+        # Required for SwiftPM to find sqlite
+        swiftpm_args += ['-Xcxx', '-I', '-Xcxx', '/usr/local/include',
+                         '-Xswiftc', '-I', '-Xswiftc', '/usr/local/include']
```

Comment:
Should SwiftPM be fixed to add these search paths on FreeBSD? My expectation is that if we don’t need them on any other Unix distributions, we shouldn’t need them here either. 

To be clear, I’m fine with this change, just want to make sure we don’t forget to do the proper solution if there is one in the future. Maybe have an issue for it?

Reply:
Thanks for filing the issue. Let’s see if there is a proper solution / what that solution is.